ui/page/livestream/view.jsx

```diff
@@ -6,11 +6,32 @@ import analytics from 'analytics';
 import LivestreamLayout from 'component/livestreamLayout';
 import moment from 'moment';
 import Page from 'component/page';
+import Yrbl from 'component/yrbl';
+import { GEOBLOCKED_CHANNELS } from 'config';
+import * as SETTINGS from 'constants/settings';
 import React from 'react';
 import { useIsMobile } from 'effects/use-screensize';
 
 const LivestreamChatLayout = lazyImport(() => import('component/livestreamChatLayout' /* webpackChunkName: "chat" */));
 
+function isLivestreamGeoAllowed(channelId: ?string, isLive: boolean) {
+  const locale: LocaleInfo = window[SETTINGS.LOCALE];
+  const geoBlock: GeoBlock = GEOBLOCKED_CHANNELS[channelId];
+
+  if (locale && geoBlock) {
+    const typeBlocked = geoBlock.types && geoBlock.types.includes('livestream');
+    const countryBlocked = geoBlock.countries && geoBlock.countries.includes(locale.country);
+    const europeanUnionOnly = geoBlock.continents && geoBlock.continents.includes('EU-UNION') && locale.is_eu_member;
+    const continentBlocked =
+      europeanUnionOnly || (geoBlock.continents && geoBlock.continents.includes(locale.continent));
+
+    return typeBlocked && !countryBlocked && !continentBlocked;
```

Comment:
adjusted this so it works for now, but eventually might need logic to allow live, but not allow other types. Not a big deal for now, probably could have just removed the check for this. But !typeblocked would have broken - would have always returned false, regardless of country/continent logic.

Reply:
Hm ... that's odd, I did test the following:
```
type="livestream"; continent="AS";  --> blocked for me
type="livestream"; continent="EU";  --> allowed for me
```

I was thinking we had to support the following configuration too as a way to quickly/temporarily disable regardless of region/country:
```
type="livestream";
  or
type="video";
```
But I guess it's probably a useless scenario to cover, lol